sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_example.ipynb

```diff
@@ -4,28 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>You could clear this output unless you think it's useful to customers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>